ResourceStatusSystem/Agent/ElementInspectorAgent.py

```diff
@@ -209,9 +209,10 @@ def _execute(self):
       except Queue.Empty:
         return S_OK()
 
-      self.log.verbose('%s ( %s / %s ) being processed' % (element['name'],
-                                                           element['status'],
-                                                           element['statusType']))
+      self.log.verbose('%s ( vO=%s / status=%s / statusType=%s ) being processed' % (element['name'],
```

Comment:
Is that really a lowercase "v"? ("vO"?). Please use "VO" as used everywhere else in the code.

Reply:
In order to match the mysql columns the columns names are capitalised. This is OK both for VO and vO, however there is also a reverse operation. This is present in ` ElementInspectoreAgent`.https://github.com/DIRACGrid/DIRAC/blob/9a70ef0a61f369db325f7fdfd859f7278d6c8d3a/ResourceStatusSystem/Agent/ElementInspectorAgent.py#L181-L184

It will lower the first character, so VO becomes vO and will break the code down the line (VO will not be included in the columns considered). It is tricky a bit, but I found naming the column vO the way out.